src/order/Order.java

```diff
@@ -0,0 +1,123 @@
+package order;
+
+import address.Address;
+import food.MenuItem;
+import restaurant.Restaurant;
+import user.Driver;
+import user.User;
+
+import java.util.ArrayList;
+
+public class Order implements Comparable<Order>{
+    private User client;
+    private Restaurant restaurant;
+    private ArrayList<MenuItem> orderItems;

```

Comment:
please use List<MenuItem> - abstract layer for references, concrete class for real object